metricbeat/include/fields/fields.go

```diff
@@ -31,5 +31,5 @@ func init() {
OB59OaNVHLCv7IdmXPO+PsR/H1oqcW29zm5KhsbmypXbaty9AgH96lh3d0dqE0V4J35C3qi2LXiQTBpPWlNHtvV1NkjQOBbBZIBz9F7rhTeRWJOEgNTQoFtcWw/+5F5KQJoTidnyQhXTswKX+TnCcRsab/q+IxifIblDa7lfpfSVj08eQUvYTKFQwpn+elPr46LCfw0PTx6dUqPzk5eTSavj09fTc+CdzcnLA6UuhsjKFBSpVlum0QMVEzC1HjP5W1jon6Aqic+UGh37paxBSqJ7RWxh9nD8R0WZCCLICzbOd4uJHaACYn1d/WNPUAbMvX3s0WQx8hM4/3SC3fLJXUiEqGtwat0XKj/OIgvXY4oQu+s+z4K/Ea+PMmOs6FpV52bCj1L6vTN15t6ZmEVobIRbHFHqFFprVcDtC0lioV9eH8tXc+U4fz8QRf2+Ul49Cv7/MD2uLTPusA7CkD42bDz374yoB18XB/5X6H5u6UosRj/Jfu+h9SSHVu+7yZqwi7w67AOyxHfDW+iDfc69N+pIbtj5j++E7xDvKkJvC47LoSb99c7a+q6VKkbhm/eXzcB6nQ1YS44tzHFaL+k7lC7sJ1/fVe1+FLhtp8Kt/cT31x+JoxXtbsibexkeZMvEeTZNxg695Ub6puAK0NXqAOWg9Q2owHUOIJl5CtDCOEzDYamwZIre7G3LTdEj3NWzUGqmmksY7UijedyVVlHTDkTkun5ok2RDOnPxWJRc5dosbsQfJS0UjNp6Wzt4PS4eX997yOeIQXJOXtY41iOIYxVdyWi/mczLiTc0om4h3MSXy+2Nm8oNHRtwfh+hNpboK3S7NKnu+yxfh6bjKj+TLaMtB99UaUsAo14L8VzKUrfXr755e3B28s31xcHF2+vD46OXx9c/uXy4PqXi75IqmUZi6SvX95vlkFfv7zv1m5SDIiVoMF8O7KWosqNVjVydyfiJfrUBcECJP7unDa9x/cX3RwBqWWZ/WlspuZJZwIy8lcAm7fUXikZtKhczoGD4QPfxaYd0APdCnMJ097KDw+evavL0qyDnZomkWrItatj85pBP7bdR/6Gio6F8Y9nc60rdf7ixXK5zJx5muXixaxmBbwA/iICFdmvLyRgMmwOL86y4/hBe1+am7C5XpQ/3IYpQ7dm8W+9wnXreqFI9dwOz5m3sYrfHWk4LpTXoHR63JnvtTLuOJuAY7tBs8ZaEFlzQjGvbEXojDKu9No8vVqWRGlWlq6lZ5tF6LLhDL9QTZjR7W3zgNTKtKvCSachjLJe8YpKy+qts96XN+e2r1oE0V+SP47HbbaKTZjrO7D/4FSupu7i65f3+/TEWdcVxzFqmH5l2Ltl7fPT05MXloP//bc/Rxz9gxb9XC0rovY8VBBG4wi0x0orrZ4ilU9TFdKoc2Go5XzsMyd9J0iUXgh5/dCTRssj5Nd3rZb+mLykzcUiNbJ3cYTHtoILEl79JcQYIsIriFEha8gfR9CiFhbRLLQLP8yiwtaiUb3zQAuqNwOnpyfpqqbTkz4pYY+r3U8HbDa1diUctz/N/nW73ohDe7JfPOpO98Si1N5jAs0Os5LfEhT327bf2MBvd5rjQ8pPeUewpDYAbup/x00N37DTf9B7McSITRAoTmGyyyYXBg5aQs1dOMFYfA8F+x1FnJNaN0+NOgdIPBHWk+UCxJzAotItXTgE+0S8HS2Ejs+5KZ5hVEPTZdy3gLSdxv+1HGrJNuL1e/HpVNLZIm5p+pCgoZBh1q9RRuhUuxKW8Q/jYO9rUa1lvh+SJ4onsU+878i1H/FfHZTORko4vKhSHbAP6llooSTRPekOr2PmqB2v0W3aqHXdpOnkL3zU85SEEu5pwBpakLC7/7sgq4PeW68lYP+i0HdpPmHYsj/0kSOiub/0o2nGyYpRa555A93SY+8esU01RWu46HmbovbHhVQ/dRy0dTfE2vgR4ytEHi9hIrS6Wxy9LdXYZdSDt0Yt9qWx/eKJFnfA2e+QuJ0XFpQ9sEpry4azoOM+HeRRmsdvj4R75pvH0eheLzL7IKayCr5aYINX80hirr82XWYxtxHDIz7R0QUSvdszF3xqGaV72WWniKHp6N9tLxzKB5tF2ZcSJPx8N1lhQXqJ0caFxAJ84tVEiqVB4mWXeXdlc0EacGoulq5+bQmTJiKFgdjubTTOqKwbwjsJeMN39trSwuGq11fuyOl4e4Ok1R7aTiPPvbd00yBs/eWZj+CK7UROtyJd0H8mLuwcnsb0wbyfmlayZloXjO+H0Ly/C8KK6nyI3Nls+uTzXXDumiB8OZdiMbAhf/eYWEfD8HY3A5GtTyN/UJ+Y4Uw8CPF3YeRhmL8HR/cxH5g5PCd5KWozje5giFtSNX1DnyQxfvBdRVFK551uVranqotC0KK4xQdum1akLgvQ3gfphXV0eplHM3wr82DdSFtXSb7lyIpiDxGFGfns0s+CqkIDcERmue2QVbAZ07QUOVCeraXNJ3a1+RpraLlyD5KrN1HDQdfnawCGYAduw8E7Dfu2Y3EP3AbJSc08N33HNmP/EHYs2wk5vaespBNWMr26/b3N4GooqNUBUKUPjvLNJFwEgAi2eGRtJ0umXKs95VMb11NUSfFPyHW7qm1DcPvNwbfhrOdeMbT8LMSsBLvT1mO3bVU3I3AdU7eMz230AtuEtjv9jf87Ady1FMUr5bo5cPY7s2fVXEh9i8rqrA1xUp7PhfT4Dppd/iTWyNpsCUcG2UmJa9uWPFLua9txhSWsnADdInUD74MPQQQXN+rDbLRJzUpNUleIP2bHmcu4zcwGXEO7+dgPbh9AyxXOhMXTMK2LS8f9eBNArvhUhIzqAsux6Gl503y+lTPDfsDDjQuVDS4A2uLb7R7ZP6pOfU8zS1FDrVQzrQSmTlssm1kTnth3QWepYKY2b/r2pa3T2+8CtnaS+zq4KLJ6340fzMBnUZCvV2/6iLgo4HE7O+GNCQjx+wqZZDMpC/f/BQAA//9xHx+o"
```

Comment:
I wonder why this has become so much longer?

Reply:
Size of global `fields.yml` files of every Beat is increased by ~1200 lines.